docker/README.md

```diff
@@ -0,0 +1,79 @@
+# Using Docker to run garage
+
+Currently there are three types of garage images:
+  - ci: used for the continuous integration of garage in TravisCI and not
```

Comment:
you appear to only have CI and headless images? also it's not clear how CI differs from headless

Reply:
A difference between the headless and CI images could be CodeClimate, since only the latter uses it, but I don't see any problem in having CodeClimate for headless. I will remove the references to the CI image.